json-type.mdx

```diff
@@ -0,0 +1,308 @@
+---
+title: 'Introducing the JSON column type'
+description: 'Use the new JSON column type for improved data storage in Xata.'
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/introducing-json-data-type/images/json-type/json-column.jpeg
+  alt: JSON type header
+author: Alejandro Martínez
+date: 09-01-2023
+published: true
+slug: json-column-type
+---
+
+Schemafull data models are great. In Xata we are convinced that it is the best way to model your data in most cases,
+but we also know that sometimes you need to store data that is not easily modeled in a relational way,
+it's not as convenient as schemaless, or you just don't want to spend time on that at an early stage.
+JSON documents as fields in a relational data store are a great way to have the best of both worlds.
+
+Many of our users have been asking for this feature, and we are happy to announce that it's finally here.
```

Comment:
If we wanted to add the link:
Many of our users have been asking for [this feature](https://xata.canny.io/feature-requests/p/json-objects)

Reply:
Added!